class-tgm-plugin-activation.php

```diff
@@ -802,25 +820,24 @@ public function notices() {
 				}
 
 				// Grab all plugin names.
-				foreach ( $message as $type => $plugin_groups ) {
-					$linked_plugin_groups = array();
-
-					// Count number of plugins in each message group to calculate singular/plural message.
-					$count = count( $plugin_groups );
+				foreach ( $message as $type => $plugin_group ) {
```

Comment:
Should be done in a separate PR, is a different issue. (and I'm preparing a number of those already, mainly focussed on abstracting (nearly) duplicate code to functions)